activestorage/lib/active_storage/attached/changes/create_many.rb

```diff
@@ -6,6 +6,7 @@ class Attached::Changes::CreateMany #:nodoc:
 
     def initialize(name, record, attachables)
       @name, @record, @attachables = name, record, Array(attachables)
+      blobs.each(&:identify_without_saving)
```

Comment:
Although `CreateOneOfMany#initialize` (inherited from `CreateOne`) also calls `identify_without_saving`, that doesn't occur until `subchanges` is materialized.  Technically, just calling `subchanges` here would suffice, but I felt writing it this way was less fragile and more self-documenting.